build/dynamic-deps.sh

```diff
@@ -65,6 +63,7 @@ _gen/
 .*_def\.py
 .*_spec\.py
 asdl/py.*
+core/completion.py
```

Comment:
still some work left to get things translating. splitting out the type checking changes to make reviewing easier